### PR TITLE
Dev mukul

### PR DESCRIPTION
**Pull request by MayurAdaptiveCode**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

About # (link your issue here)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [x] I have added screenshots related to my pull request (for frontend tasks)
- [x] I have pasted a gif showing the feature.
- [x] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/pages/App.tsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ function App(): JSX.Element {
                         ref={bodyRef}
                         className="flex flex-col items-center justify-between flex-grow h-full overflow-y-auto overflow-x-hidden z-0 pt-4 sm:pt-8 px-4 md:pt-10 pb-4"
                         style={{
-                            background: theme.color.mainBg
+                            background: 'white'
```
Comment (sirpy) [inline src/pages/App.tsx:115]: why not using theme?

Comment (MayurAdaptiveCode) [reply src/pages/App.tsx:115]: we had a type issue with the theme object so for now decided to ignore and fix later

Comment (sirpy) [reply src/pages/App.tsx:115]: ok but how would you know you need to get back to this specific place?
you must mark it with a TODO or something

Comment (MayurAdaptiveCode) [reply src/pages/App.tsx:115]: OK, will update this in the next feature branch

**Pull request merged**